examples/consumer/main.go

```diff
@@ -22,11 +22,11 @@ func main() {
 	// register consumer
 	csm := beanq.NewConsumer()
 	// register normal consumer
-	csm.Register("g2", "ch2", func(task *beanq.Task) error {
-		// TODO:logic
-		beanq.Logger.Info(task.Payload())
-		return nil
-	})
+	// csm.Register("g2", "ch2", func(task *beanq.Task) error {
+	// 	// TODO:logic
+	// 	beanq.Logger.Info(task.Payload())
+	// 	return nil
+	// })
```

Comment:
If you don't need this code then remove it OR you can create a separate example and keep the code there

Reply:
ok.